financial-connections/src/main/java/com/stripe/android/financialconnections/features/success/SuccessContent.kt

```diff
@@ -369,7 +367,7 @@ private fun calculateBodyHeightForPreview(config: Configuration, state: SuccessS
     // We need to manually calculate this for our screenshot tests, as we've been unable to
     // delay the capture until the offset animation finishes.
     val isPhone = config.orientation == Configuration.ORIENTATION_PORTRAIT
-    return if (state.payload()?.customSuccessMessage != null && isPhone) {
```

Comment:
This check is not useful anymore for custom vs non-custom messaging (I believe this was used to make the preview look ok with 1-line vs multi-line states?) Messages in payload are always present as they're resolved in the viewmodel, so we can't make this distinction on the view.